include/superior_mysqlpp/uncaught_exception_counter.hpp

```diff
@@ -4,32 +4,19 @@
 
 #pragma once
 
-#include <cxxabi.h>
+#include <exception>
 
 namespace SuperiorMySqlpp
 {
-    inline int uncaughtExceptions() noexcept
-    {
-        auto* globalsPtr = __cxxabiv1::__cxa_get_globals();
-        auto* globalsEreasedPtr = reinterpret_cast<char*>(globalsPtr);
-        auto* uncaughtExceptionsErasedPtr = globalsEreasedPtr + sizeof(void*);
-        auto* uncaughtExceptionsPtr = reinterpret_cast<int*>(uncaughtExceptionsErasedPtr);
-        return *uncaughtExceptionsPtr;
-    }
-
     class UncaughtExceptionCounter
```

Comment:
I'd rename this to something like `MultipleExceptionDetector` + method then doesn't even need a name, perhaps just operator().

Reply:
I think that name should be some predicate (`MultipleExceptionsOccured`, for example) If it should have only `operator()`.